types/hand-crafted/modules/RTC/JitsiLocalTrack.d.ts

```diff
@@ -5,9 +5,9 @@ export default class JitsiLocalTrack extends JitsiTrack {
   constructor( trackInfo: { rtcId: number, stream: unknown, track: unknown, mediaType: unknown, videoType: unknown, effects: unknown, resolution: unknown, deviceId: string, facingMode: CameraFacingMode, sourceId: unknown } ) // TODO:
   isEnded: () => boolean;
   setEffect: ( effect: unknown ) => Promise<unknown>; // TODO:
-  mute: () => void;
-  unmute: () => void;
-  dispose: () => void;
+  mute: () => Promise<unknown>;
+  unmute: () => Promise<unknown>;
+  dispose: () => Promise<unknown>;
```

Comment:
Why unknown?

Reply:
I used `void` for the dispose promise now. I have no idea what the overs return.